apps/browser/qml/pages/components/ToolBar.qml

```diff
@@ -139,10 +139,26 @@ Column {
             Browser.TabButton {
                 id: tabs
 
-                icon.source: webView.privateMode ? "image://theme/icon-m-incognito" : "image://theme/icon-m-tabs"
-                opacity: secondaryToolsActive || findInPageActive ? 0.0 : 1.0
+                icon.source: {
+                    if (webView.privateMode) {
+                        return webView.tabModel.count > 0 ? "image://theme/icon-m-incognito-selected"
+                                                          : "image://theme/icon-m-incognito"
+                    } else {
+                        return "image://theme/icon-m-tabs"
+                    }
+                }
+
+                label.color: {
+                    if (webView.privateMode) {
+                        return Theme.overlayBackgroundColor ? Theme.overlayBackgroundColor : "black"
+                    } else {
+                        return down ? Theme.highlightColor : Theme.primaryColor
```

Comment:
```suggestion
                        return highlighted ? Theme.highlightColor : Theme.primaryColor
```

Whilst end result is the same, I find highlighted usage better when coloring.

Reply:
Thanks; I've made the change.